contrib/backporting/start-backport

```diff
@@ -33,8 +33,9 @@ BRANCH=$(echo "$BRANCH" | sed 's/^v//')
 # have the same conflicting branch name.
 SUFFIX="${2}"
 
-git fetch origin
-if ! git branch -a | grep -q "origin/v$BRANCH$" ; then
+REMOTE=$(get_remote)
```

Comment:
Hm. I'm a bit torn here.

Firstly, this script here has an optional suffix argument, so is likely a bit more complicated to add than a simple if statement. Secondly, `cherry-pick` and `check-stable` also unconditionally invoke `get_remote()`, so even if a user needs set a custom remote for `start-backport`, they will likely fail in subsequent steps.

I given this, I'd actually rather consider removing the positional argument from `submit-backport` instead.

Reply:
Yeah, I will remove the optional upstream remote setting from `submit-backport` again. I don't see a reason why it would needed, considering all other scripts also work fine without it being configurable. If there is a use-case for it to be configurable, it needs to be in a matter that works for all four scripts, not just one of them.

Also, I noticed a small bug in `submit-backport` (it refers to `$ORIGIN` instead of `$UPSTREM_REMOTE`). Will rework the PR to address that.